src/components/app.js

```diff
@@ -212,11 +212,20 @@ const App = () => {
         connectNotificationsUpdateConfig,
     ]);
 
+    const updateRootDirectories = () => {
+        fetchRootFolders().then((data) => {
+            let sortedData = [...data];
+            sortedData.sort(function (a, b) {
```

Comment:
Why do we need a sort ? Is directory Tree component not a better place to sort those root directories ?

Reply:
we need to sort root directories to keep trees in the same order once we delete or insert a new root directory

Reply:
ok